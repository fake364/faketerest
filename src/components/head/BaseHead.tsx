import React from 'react';
import COMMON_CONSTANTS from '../../common/constants/commons';
import Head from 'next/head';

type Props = {};

const BaseHead: React.FC<Props> = ({ children }) => (
	<Head>
		<title>{COMMON_CONSTANTS.TITLE}</title>
		<link rel="icon" href="/favicon.ico" />
		{children}
	</Head>
);

export default BaseHead;
