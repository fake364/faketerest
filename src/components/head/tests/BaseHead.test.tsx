import { mount } from 'enzyme';
import { expect } from '@jest/globals';
import React from 'react';
import BaseHead from '../BaseHead';

describe('BaseHead', () => {
  it('should render head container properly', () => {
    const component = mount(<BaseHead>Test</BaseHead>);
    expect(component).toMatchSnapshot();
  });
});
