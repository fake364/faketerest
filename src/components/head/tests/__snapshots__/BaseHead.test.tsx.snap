// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseHead should render head container properly 1`] = `
<BaseHead>
  <Head>
    <SideEffect
      headManager={
        Object {
          "_pendingUpdate": null,
        }
      }
      inAmpMode={false}
      reduceComponentsToState={[Function]}
    />
  </Head>
</BaseHead>
`;
